src/components/structures/RoomView.tsx

```diff
@@ -1652,6 +1663,23 @@ export default class RoomView extends React.Component<IProps, IState> {
         if (auxPanelMaxHeight < 50) auxPanelMaxHeight = 50;
 
         this.setState({auxPanelMaxHeight: auxPanelMaxHeight});
+
+        // Let the bubble layout choose between single side and both sides by threshold
+        if (this.state.layout == Layout.Bubble && this.state.adaptiveSideBubbles && this.roomView.current) {
+            // ToDo: Find better way to get the current width (references, but which???)
+            const messagelists = this.roomView.current.getElementsByClassName("mx_RoomView_MessageList");
+            let width = 0;
+            for (let i = 0; i < messagelists.length; i++) {
+                const boundingBox = messagelists[i].getBoundingClientRect();
+                if (boundingBox.width > width) width = boundingBox.width;
+            }
```

Comment:
```suggestion
            for (const messageList of messagelists) {
                const boundingBox = messagelist.getBoundingClientRect();
                if (boundingBox.width > width) width = boundingBox.width;
            }
```

This mainly a personal preference, so feel free to ignore

Reply:
I'm not sure this is the optimal solution anyway. Here I'm iterating over the "normal" message list and the one of the search. Thus the iteration. I feel this can be solved better (more Reacty) with some sort of references or the like. However, I couldn't figure out a working solution although I struggled quite some time with this.
I'd like some input here. **I** definitely can't solve that in a better way than I did.

Reply:
From a quick look, I can't see an easy way to fix this either. I'll try to think about it some more though I am not sure it has a simple solution

Reply:
Sorry, why is it that we can't just use refs?

Reply:
I didn't get them working. I need one to the search results panel's and the normal message list.

Reply:
Ah, I'll try to play with them a bit too, if I get time, to see if I run into the same problem

Reply:
I'm pretty sure it _can_ be solved better for with refs or the like.